spec/requests/posts_spec.rb

```diff
@@ -2,6 +2,26 @@
 
 RSpec.describe 'Posts', type: :request do
   describe 'GET /index' do
```

Comment:
- [ ] [REQUIRED] Good job so far! Kindly on my side on your `posts_spec.rb` file's `GET/index` one of the test failed, please fix it because all tests should be passed 🙏🏼 

![image](https://github.com/AlyAtalla/blog-app/assets/63915024/d3927be3-30a5-44ec-bb36-ff2fb5aaf77c)